rmw_opendds_cpp/src/rmw_node.cpp

```diff
@@ -26,20 +26,18 @@ rmw_node_t *
 rmw_create_node(
   rmw_context_t * context,
   const char * name,
-  const char * namespace_,
-  size_t domain_id,
-  bool localhost_only)
+  const char * name_space,
+  size_t /* domain_id */,
+  bool /* localhost_only */)
 {
-  (void)domain_id;
-  (void)localhost_only;
   RMW_CHECK_FOR_NULL_WITH_MSG(context, "context is null", NULL);
   if (!check_impl_id(context->implementation_identifier)) {
     return NULL;
   }
   RMW_CHECK_FOR_NULL_WITH_MSG(context->impl, "context->impl is null", NULL);
   RMW_CHECK_FOR_NULL_WITH_MSG(name, "node name is null", NULL);
-  RMW_CHECK_FOR_NULL_WITH_MSG(namespace_, "node namespace_ is null", NULL);
-  return create_node(*context, name, namespace_);
+  RMW_CHECK_FOR_NULL_WITH_MSG(name_space, "node name_space is null", NULL);
+  return create_node(*context, name, name_space);
```

Comment:
The following is from the review of PR#56 on the same issue:
@sonndinh
Suggested change:
- create_node(rmw_context_t & context, const char * name, const char * namespace_);
+ create_node(rmw_context_t & context, const char * name, const char * namespace);
 
@lij-oci
I don't like namespace_ as a parameter either, since underscore suffix is usually used for member variables. So I changed it to name_space in the .cpp file, but forgot it here in .hpp. I understand the original namespace_ is to avoid collision with the cpp keyword namespace.
 
@sonndinh
Yeah, name_space is better, I forgot namespace is a keyword.

Reply:
name_space is replaced with namespace_ for consistency with rmw.h.